llvm/utils/TableGen/X86InstrMappingEmitter.cpp

```diff
@@ -335,6 +337,31 @@ void X86InstrMappingEmitter::emitND2NonNDTable(
   printTable(Table, "X86ND2NonNDTable", "GET_X86_ND2NONND_TABLE", OS);
 }
 
+// Method emitSSE2AVXTable will create table GET_X86_SSE2AVX_TABLE for SSE to
+// AVX instruction mapping in X86GenInstrMapping.inc file, In table first entry
+// will be SSE instruction and second entry will be equivalent AVX instruction
+// Example:-  "{ X86::ADDPDrm, X86::VADDPDrm },"
```

Comment:
It's obvious from the name. We can remove this comment.

Reply:
Done